tmp/build/patch-charts.sh

```diff
@@ -250,6 +250,22 @@ function patchTemplates() {
     sed -i -e 's/\/health/\/tmp\/health/' ${HELM_DIR}/istio/charts/sidecarInjectorWebhook/templates/deployment.yaml
   fi
 
+  # add oauth-proxy default hub, image, tag and imagePullPolicy
+  if [[ "${COMMUNITY,,}" == "true" ]]; then
+    sed -i -e '/^global:/a \
+\  oauthproxy:\
+\    hub: docker.io/openshift\
+\    image: oauth-proxy\
+\    tag: latest\
```

Comment:
We need to find an alternative to the latest tag

Reply:
@dgn  There is `quay.io/openshift/origin-oauth-proxy:4.1` which is open to anyone.
Not sure if it's the same bits as on docker hub.
We have not tested it. You should probably test it before using.

Reply:
@jwendell testing that image now